website/docs/Installation/linux-install.md

```diff
@@ -1,64 +1,131 @@
+## Installing Podman Desktop from Flathub {#flathub}
 
-### 1. Using [FlatHub](https://flathub.org/apps/details/io.podman_desktop.PodmanDesktop)
+Consider installing the Podman Desktop from Flathub to have:
 
-With FlatHub, you can install Podman Desktop with just a single command.
+* One command installation
+* Package updates
 
-> Follow the [setup guide](https://flatpak.org/setup/) before installing Podman Desktop
+#### Prerequisites
 
-```sh
-flatpak install --user flathub io.podman_desktop.PodmanDesktop
-```
-After installation, to open Podman Desktop, use the following command:
+* [Flatpak](https://flatpak.org/setup/)
+* [Podman](https://podman.io/whatis.html) stable version
 
-```sh
-flatpak run io.podman_desktop.PodmanDesktop
-```
+#### Procedure
 
-### 2. Using [Flatpak](https://flatpak.org/setup/) application from [Downloads](/downloads/linux)
+1. Assert the Flathub repository is enabled, and add it if required:
 
-:::infoPrerequisite 
-Verify Flatpak is set up in your distribution. Click [here](https://flatpak.org/setup/) to learn more about setting up Flatpak.
-:::
+    ```shell
+    $ flatpak remote-add --if-not-exists --user flathub https://flathub.org/repo/flathub.flatpakrepo
+    ```
 
-Open the Terminal and go to Downloads directory. 
+2. Install Podman Desktop from Flathub:
 
-```sh
-cd Downloads
-```
+    ```shell
+    $ flatpak install --user flathub io.podman_desktop.PodmanDesktop
+    ```
 
-> The Flatpak identifier for the application is `io.podman_desktop.PodmanDesktop`. This is required to install & start the application using Flatpak.
+#### Verification
 
-Run the following command to install the Flatpak application,
+* Open Podman Desktop from a terminal:
 
-```sh
-flatpak install io.podman_desktop.PodmanDesktop
-```
+  ```shell
+  $ flatpak run io.podman_desktop.PodmanDesktop
+  ```
 
-Run the following command to start the application,
+#### Update
 
-```sh
-flatpak run io.podman_desktop.PodmanDesktop
-```
+* Update Podman Desktop from Flathub:
 
-You can read more about running Flatpak applications [here](https://docs.flatpak.org/en/latest/using-flatpak.html).
+  ```shell
+  $ flatpak update --user io.podman_desktop.PodmanDesktop
+  ```
 
-### 3. Using zip file
+#### Additional resources
 
-Download the zip file and extract the folder.
+* [Podman Desktop Flathub package](https://flathub.org/apps/details/io.podman_desktop.PodmanDesktop)
+* [Using Flatpak](https://docs.flatpak.org/en/latest/using-flatpak.html)
 
-Navigate within the folder and double-click on the `podman-desktop` executable file. This should start the application for you.
+#### Next steps
 
-## Next Steps
+* [Getting started](../getting-started/getting-started)
 
-Learn more on how to get started with Podman Desktop by clicking [here](/docs/getting-started/getting-started).
+## Installing Podman Desktop from a Flatpak bundle {#flatpak-bundle}
+
+Consider installing a Flatpak bundle rather than Flathub when:
+
+* You cannot use Flathub.
```

Comment:
```suggestion
* You cannot use Flathub
```

Reply:
Punctuation in lists is complex. Usually no periods, but:

> Write lists so that all or none of the items start with complete sentences. If list items comprise only complete sentences, include a period after each sentence.

(IBM style guide)